Used consistent pod template naming in Kubernetes Executor documentation (#384)
astronomer/docs: enterprise/v0.23/05_customize-airflow/04_kubernetes-executor.md

If you want to use a pod template for all DAGs within a given Airflow Deployment, you can add the pod template to the Deployment as an environment variable. To do so:

> 1. Update your Dockerfile to copy your customized pod template into your Docker image. For instance, if your customized pod template file name is `new_pod_template.yaml`, you would add the following line to your Dockerfile:

Reviewer: ```suggestion
1. Add a command to your Dockerfile that copies your customized pod template into your Docker image. For instance, if your customized pod template file name is `new_pod_template.yaml`, you would add the following line to your Dockerfile:
```

    ```

>     COPY new_pod_template.yaml /tmp/copied_pod_template.yaml

Reviewer: @jwitz I'm following the creation of `new_pod_template.yaml`, but not so much `copied_pod_template`. In this part of the doc, is copied referring to `pod_templates/pod_template_file.yaml`? If so, should it just be referenced as that? If not, where is it coming from?

Author: Copied pod template is being created in this command. We're basically creating a containerized version of the file in that tmp folder. I used copied in the name to distinguish between the two copies, since you specifically have to reference the containerized one later in this setup. Open to chaning the actual naming tho

Reviewer: Got it, makes sense. I just tweaked the wording here to make it more clear - lmk what you think!

>     ```

Reviewer: ```suggestion
    ```
    This command uses `new_pod_template.yaml` to create `copied_pod_template.yaml` at build time as part of your Docker image. You'll specify this file in an Environment Variable in Step 2.

```
